addons/web/static/src/legacy/js/views/basic/basic_model.js

```diff
@@ -1589,7 +1589,16 @@ var BasicModel = AbstractModel.extend({
         options = options || {};
         record._changes = record._changes || {};
         if (!options.doNotSetDirty) {
-            record._isDirty = true;
+            // Check if the changes can dirty.
+            for (const fieldName of Object.keys(changes)) {
+                const fieldInfo = Object.values(record.fieldsInfo).find(el => el[fieldName]);
+                const fieldNameInfo = fieldInfo && fieldInfo[fieldName];
+                const fieldNameInfoOptions = fieldNameInfo && fieldNameInfo.options;
+                const changeCanDirty = !(fieldNameInfoOptions && fieldNameInfoOptions.no_dirty);
+                if (changeCanDirty) {
+                    record._isDirty = true;
+                }
+            }
```

Comment:
@odoo/rd-framework-js Hi, do you want to have a look here or do you want us to take care of the review?
This is following https://discord.com/channels/678381219515465750/687338039717920792/1118919199788638230
Note that it's the only real diff of the PR, the rest is a test.

Reply:
We will discuss it Monday via discord call to expose the problem and see what we do :+1: